extensions/amp-viewer-integration/0.1/highlight-handler.js

```diff
@@ -48,10 +56,20 @@ const NUM_ALL_CHARS_LIMIT = 1500;
 
 /**
  * TextRange represents a text range.
- * @typedef {{sentences: !Array<string>}}
+ * If doNothing is true, checks whether highlighted texts are found but do not
+ * show highlighted texts to users.
+ * @typedef {{sentences: !Array<string>, doNothing: boolean}}
```

Comment:
What's the purpose of checking without highlighting?

Reply:
Replied privately.

Reply:
Please rename to `skipRendering`.

Reply:
Done